src/controllers/signAccountOp/signAccountOp.ts

```diff
@@ -814,6 +814,7 @@ export class SignAccountOpController extends EventEmitter {
         isErc4337Broadcast(this.#network, accountState),
       isGasTank: this.feeTokenResult.flags.onGasTank,
       inToken: this.feeTokenResult.address,
+      networkId: this.feeTokenResult.networkId,
```

Comment:
`networkId` is not the best name as it's not clear what it indicates. Especially in this case when it's about the fee token. Let's rename it to `feeTokenNetworkId`